test/functional/admin/statistics_announcement_publications_controller_test.rb

```diff
@@ -25,6 +25,12 @@ class Admin::StatisticsAnnouncementPublicationsControllerTest < ActionController
     refute_select ".govuk-table"
   end
 
+  view_test "GET :index shows inset text" do
+    get :index, params: { statistics_announcement_id: @official_statistics_announcement }
+
+    assert_select ".govuk-inset-text", text: "The type of the document you are trying to connect must match the type of the statistics announcement. Only Official Statistics documents will be shown."
```

Comment:
I've lost all sense of the English language, help.

Reply:
LGTM 👍